OERCONE/iocBoot/iocOERCONE-IOC-01/st.lua

```diff
@@ -0,0 +1,21 @@
+-- #!../../bin/windows-x64/OERCONE-IOC-01
+
+-- # You may have to change OERCONE-IOC-01 to something else
+-- # everywhere it appears in this file
+
+-- # Increase this if you get <<TRUNCATED>> or discarded messages warnings in your errlog output
+iocsh.errlogInit2(65536, 256)
+
+-- # envPaths
+iocsh.iocshLoad(string.format("%s/dbload.cmd", iocstartup))
+
+-- # cd ${TOP}
```

Comment:
Remove commented out code

Reply:
The st.lua file is never called so was dead code.
Removed the file.